upflux-client/src/features/updateManagement/UpdateManagement.tsx

```diff
@@ -0,0 +1,137 @@
+  const pendingMachines = machines.length - updatedMachines;
+
+  // Chart data
+  const chartData = [
+    { name: "Updated", value: updatedMachines, color: "#007bff" },
+    { name: "Pending Updates", value: pendingMachines, color: "#ff0000" },
+  ];
+  
+
+//   // Chart options
+// const chartOptions = {
+//     cutout: '60%',
+//     plugins: {
+//       legend: {
+//         display: false, // Disable the default legend
+//       },
+//     },
+//   };
+
+  return (
+    <Stack className="update-management-content">
+      {/* Header */}
+      <Box className="header">
+        <Text size="xl" fw={700}>
+          Update Management
+        </Text>
+      </Box>
+
+      <Box className="content-wrapper">
+        {/* Overview Section */}
+        <Group className="overview-section">
+          {/* Chart Section */}
+          <Box className="chart">
+            <DonutChart  className="chart" data={chartData} />;
+            {/* Custom Text in the center of the doughnut */}
+            <Text className="chart-text">
+              Machines <br /> <br /> {machines.length}
+            </Text>
+          </Box>
+
+          {/* Legend */}
+          <Stack className="legend">
+            <Group className="legend-item">
+              <Box className="circle blue"></Box>
+              <Text size="sm">{updatedMachines} Updated</Text>
+            </Group>
+            <Group className="legend-item">
+              <Box className="circle red"></Box>
+              <Text size="sm">{pendingMachines} Pending Updates</Text>
+            </Group>
+          </Stack>
+
+          {/* Action Buttons */}
+          <Stack className="button-group">
+            <Button className="configure-button">Configure Update</Button>
+            <Button className="smart-button">Smart Update</Button>
+          </Stack>
+        </Group>
+
+        {/* Table Section */}
+        <Box>
+          <Table>
+            <thead>
+              <tr>
+                <th>Machine ID</th>
+                <th>Last Update</th>
+                <th>Current Status</th>
+                <th>View</th>
+              </tr>
+            </thead>
+            <tbody>
+              {machines.map((machine) => (
+                <tr key={machine.id}>
+                  <td>{machine.id}</td>
+                  <td>{machine.lastUpdate}</td>
+                  <td>
+                    <Badge
+                      color={
+                        machine.status === "Alive"
+                          ? "green"
+                          : machine.status === "Shutdown"
+                          ? "red"
+                          : "gray"
+                      }
+                    >
+                      {machine.status}
+                    </Badge>
+                  </td>
+                  <td>
+                    <img src={view} alt="view" className="view" />
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </Table>
```

Comment:
Why are we not using mantine table component

Reply:
Thansks, that is something that I missed. I have updated it now to use the table component from Mantine.